### ISSUE TITLE
[Skia macOS] Scrolling by mouse wheel is very slow

### ISSUE THREAD

**Issue by MartinZikmund** [unoplatform/uno#16400]
### Current behavior

When mouse wheel is scrolled, it scroll bar moves very very slowly

### Expected behavior

Should be on par with other macOS apps

### Works on UWP/WinUI

None

### Affected platforms

Skia (GTK on Linux/macOS/Windows)

**Comment by spouliot** [comment]
I do not have an issue with scrolling with both the laptop trackpad or with a Magic Mouse. In both cases the speed of scrolling seems correct.

@MartinZikmund Not sure if that was fixed or if you were using a mouse with a wheel ? I can try to find one... somewhere...

**Comment by MartinZikmund** [comment]
Let me try again 👍

**Comment by MartinZikmund** [comment]
@spouliot it does seem that the mouse scroll wheel is the problem, scrolling it just goes line by line, very slowly. I can try some change on my side if you point me in the right direction 😁

**Comment by spouliot** [comment]
@MartinZikmund what kind of mouse are you using ? it's not a Magic Mouse right ?

> I can try some change on my side if you point me in the right direction 😁

If you build `SampleApp.Skia.Generic.csproj` locally (on a Mac) the native library will be built with `DEBUG` enabled - which will log a lot of data on stdout.

Some mouse events are commented out (there's too much of them) but can be re-enabled locally.

https://github.com/unoplatform/uno/blob/46b77257597437ac8690015e584a8a9bc398d2fb/src/Uno.UI.Runtime.Skia.MacOS/UnoNativeMac/UnoNativeMac/UNOWindow.m#L808

**Comment by spouliot** [comment]
k, found an old mouse with a scroll wheel and I can duplicate the issue.
